R/qsys_lsf.r

```diff
@@ -10,8 +10,8 @@ LSF = R6::R6Class("LSF",
             private$set_common_data(fun=fun, const=const, seed=seed)
         },
 
-        submit_job = function(memory=NULL, log_worker=FALSE) {
-            values = super$submit_job(memory=memory, log_worker=log_worker)
+        submit_job = function(memory=NULL, walltime=NA, log_worker=FALSE) {
+            values = super$submit_job(memory=memory,walltime=walltime, log_worker=log_worker)
```

Comment:
please adhere to the function call style of the rest of the package (space after comma)